x-pack/plugins/apm/public/services/rest/ml.ts

```diff
@@ -68,24 +69,22 @@ export interface MLJobApiResponse {
   count: number;
   jobs: Array<{
     job_id: string;
-    [key: string]: any;
   }>;
 }
 
 export async function getMLJob({
   serviceName,
-  transactionType,
-  anomalyName = 'high_mean_response_time'
```

Comment:
Following the law of YAGNI. If we need this flexibility later, we can easily added it to the ml utility methods.